pages/k8s/cni-canal.md

```diff
@@ -0,0 +1,72 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "CNI with canal"
+  description: How to manage and deploy Kubernetes with Canal
+keywords: CNI, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: cni-canal.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+'Canal' is a shorthand for saying "Calico and Flannel", a common practise which sets up Calico to handle policy management and Flannel to manage the network itself. The manual setup is explained in the [Calico documentation][canal].  
```

Comment:
I would omit the last sentence that links to https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/flannel. It might mislead someone into thinking they can manually deploy canal into CDK using that method. That's not possible with the current implementation of charms.

If the intro feels too short without it, here's some fluff to consider:

> 'Canal' is a shorthand for saying "Calico and Flannel", a common practise which sets up Calico to handle policy management and Flannel to manage the network itself. This combination brings in Calico's support for the [NetworkPolicy] feature of Kubernetes, while utilizing Flannel's UDP-based network traffic to provide for an easier setup experience that works in a wider variety of host network environments without special configuration.

[NetworkPolicy]: https://kubernetes.io/docs/concepts/services-networking/network-policies/

On a more nitpicky note, the mix of single and double quotes in the first sentence looks a bit odd to me.

Reply:
:+1: